scripts/example_script.py

```diff
@@ -0,0 +1,238 @@
+            tempImageId = item['_id']
+            print('image name %s ' % (item['name']))
+            img_list.append(tempImageId)
+            make_large_image(gc, tempImageId)
+
+    # now create PATIENT folders inside our new folder
+
+    patientFolderList = []
+    patient_folder_name = 'Patient%d'
+
+    for i in range(1, 4):
+        tempName = patient_folder_name % (i)
+
+        patientFolderList.append(
+                gc.load_or_create_folder(tempName, newFolderId, 'folder'))
+
+    # now let us move an image into the PATIENT1 folder
+
+    patient1Folder = patientFolderList[0]
+    patient1FolderId = patient1Folder['_id']
+
+    srcImageId = img_list[0]
+
+    moveParams = {
+        'resources': json.dumps({
+            'item': [str(srcImageId)]}),
+        'parentType': 'folder',
+        'parentId': patient1FolderId
+    }
+
+    try:
+        gc.put('resource/move', moveParams)
+    except HttpError as err:
+        print_http_error(err)
+        print ('exited')
+        exit()
+
+    # now let us edit the PATIENT1 folder
+
+    patient1MetaData = {'PatientAge': 22, 'PatientDeceased': True}
+
+    try:
+        gc.addMetadataToFolder(patient1FolderId, patient1MetaData)
+    except HttpError as err:
+        print_http_error(err)
+        print ('exited')
+        exit()
+
+    # let us add meta data to the image we moved
+    # note how the id remains the same
+    newSrcImageMetaData = {'SlideSource': 'FromTCGA', 'SlideType': 'DX'}
+    try:
+        gc.addMetadataToItem(srcImageId, newSrcImageMetaData)
+    except HttpError as err:
+        print_http_error(err)
+        print ('exited')
+        exit()
+
+    # find all folders determine if whether each folder is empty or not
+    allFolders = []
+
+    allFolders = recurse_get_resource(gc, collectionId,
+                                      'folder', 'collection')
+
+    # if the folder size is zero then delete it
+    for folder in allFolders:
+
+        if folder['size'] == 0 and isEmpty(gc, folder['_id']):
+            try:
+                print('will delete folder %s' % folder['name'])
+
+                gc.delete('folder/%s' % folder['_id'])
+
+            except HttpError as err:
+                print_http_error(err)
+                print('could not delete folder %s' % (folder['name']))
+                print ('exited')
+                exit()
+    # already have a list of all items under the designated collection
+    # check whether a file under an item is a README.txt
+    # if so delete the entire item
+    for item in itemList:
+        fileIdList = gc.listFile(item['_id'])
+        for girder_file in fileIdList:
+            if 'README.txt' in girder_file['name']:
+                print('deleting item %s' % item['name'])
+                try:
+                    gc.delete('item/%s' % item['_id'])
+                except HttpError as err:
+                    print_http_error(err)
+                    print('could not delete item %s' % (item['name']))
```

Comment:
Shouldn't need the `try` block here, since `main` is wrapped in one. I don't think it's that important to have the extra `print` statement on an error here.